tests/utils.py

```diff
@@ -180,3 +180,54 @@ def set_dict_element(data, path, value, delimiter='.'):
     path, _, key = path.rpartition(delimiter)
     (get_dict_element(data, path) if path else data)[key] = value
     return data
+
+
+def get_grain(host, key):
+    with host.sudo():
+        output = host.check_output(
+            'salt-call --local --out=json grains.get "{}"'.format(key)
+        )
+        grain = json.loads(output)['local']
+
+    return grain
+
+
+class PrometheusApiError(Exception):
+    pass
+
+
+class PrometheusApi():
```

Comment:
(nit) No need for the parentheses

Reply:
I disagree to me you should add `object`, this one is old style Class pretty sure pylint or flake8 say something about this one

Reply:
Guillaume is right, in Python 3 you don't need to inherit from object explicitly.
old-style class are a thing of the past (Python 2).

Reply:
Who are you calling old?! :older_man:

Reply:
Hmm ok you right, we use python3 in pytest

Reply:
> old-style class are a thing of the past (Python 2).

Python 2 is a thing of the past :trollface: